sawtooth-sdk-transaction-processor/src/main/java/sawtooth/sdk/processor/TransactionProcessor.java

```diff
@@ -201,6 +204,7 @@ private TransactionHandler findHandler(final Message message) {
   private void submitTaskForMessage(final Message message) {
     TransactionHandler handler = findHandler(message);
     if (handler == null) {
+      // No handler available for this message, noop
```

Comment:
Seems like this is in the wrong commit - which ever commit it is in should log the receipt of an unhandlable message, as well.

Reply:
Yeah agreed, the original just returned silently. seems like there should be a warning at least.